test/groovy/DockerExecuteTest.groovy

```diff
@@ -1,61 +1,97 @@
 
 import org.junit.Before
+import org.junit.Rule
 import org.junit.Test
 
+import com.lesfurets.jenkins.unit.BasePipelineTest
+
+import util.JenkinsConfigRule
+import util.JenkinsLoggingRule
+import util.JenkinsSetupRule
+
 import static org.junit.Assert.assertEquals
 import static org.junit.Assert.assertTrue
 import static org.junit.Assert.assertFalse
 
-class DockerExecuteTest extends PiperTestBase {
+class DockerExecuteTest extends BasePipelineTest {
     private DockerMock docker
 
-    String echos
+    @Rule
+    public JenkinsSetupRule jsr = new JenkinsSetupRule(this)
+
+    @Rule
+    public JenkinsLoggingRule jlr = new JenkinsLoggingRule(this)
+
+    @Rule
+    public JenkinsConfigRule jcr = new JenkinsConfigRule(this)
 
     int whichDockerReturnValue = 0
 
+    def cpe
+    def examinee;
+
     @Before
-    void setUp() {
-        super.setUp()
+    void init() {
 
         docker = new DockerMock()
         binding.setVariable('docker', docker)
         binding.setVariable('Jenkins', [instance: [pluginManager: [plugins: [new PluginMock()]]]])
 
-        echos = ''
-        helper.registerAllowedMethod("echo", [String.class], { String s -> echos += " $s" })
         helper.registerAllowedMethod('sh', [Map.class], {return whichDockerReturnValue})
+
+        prepareObjectInterceptors(this)
+
+        cpe = loadScript('commonPipelineEnvironment.groovy').commonPipelineEnvironment
+        examinee = loadScript('dockerExecute.groovy').dockerExecute
```

Comment:
I would suggest to call it with the name of the script to be loaded appended with `Script` instead of `examinee`.

In this case it would be `dockerExecuteScript`

This applies to all tests

Reply:
done

IMO "examinee" is good understandable from the test context. But <xxx>Script is also fine. I dislike to fight on name as long as they are provide an acceptable level of understandablity.